tests/src/test/scala/com/nvidia/spark/rapids/unit/DecimalUnitTest.scala

```diff
@@ -21,8 +21,9 @@ import java.math.RoundingMode
 import scala.util.Random
 
 import ai.rapids.cudf.{ColumnVector, DType, HostColumnVector}
-import com.nvidia.spark.rapids.{GpuAlias, GpuBatchScanExec, GpuColumnVector, GpuIsNotNull, GpuIsNull, GpuLiteral, GpuOverrides, GpuScalar, GpuUnitTests, HostColumnarToGpu, RapidsConf}
+import com.nvidia.spark.rapids._
 
+import org.scalatest.Matchers.convertToAnyShouldWrapper
 import org.apache.spark.SparkConf
```

Comment:
Could we totally revert all of the changes to this file, as it looks like they are not needed.